docs/content/guides/dagster/asset-versioning-and-caching.mdx

```diff
@@ -156,17 +173,17 @@ width={2661}
 height={1748}
 />
 
-Click "Materialize stale and missing" to get both assets up-to-date again.
+Click **Materialize stale and missing** to get both assets up-to-date again.
 
 ---
 
-## Step Three: Computing your own data versions
+## Step three: Computing your own data versions
 
-A data version is like a fingerprint for the value that an asset represents, i.e. the output of its materialization function. We therefore want our data versions to correspond 1-1 to the possible return values of a materialization function. Dagster auto-generates data versions by hashing the code version together with input data versions. This satisfies the above criterion in many cases, but sometimes a different approach is necessary.
+A data version is like a fingerprint for the value that an asset represents, i.e. the output of its materialization function. Therefore, we want our data versions to correspond o the possible return values of a materialization function. Dagster auto-generates data versions by hashing the code version together with input data versions. This satisfies the above criterion in many cases, but sometimes a different approach is necessary.
```

Comment:
>to correspond o the possible return values

Typo. Not sure what you were intending here but we need to retain the 1-1 concept-- maybe "one-to-one"?